src/pizzeria/OrderItem.java

```diff
@@ -1,4 +1,50 @@
 package pizzeria;
 
 public class OrderItem {
+
+    private Pizza pizza;
+    private int pizzaCount;
+
+    public OrderItem (Pizza pizza, int pizzaCount){
+        this.pizza = pizza;
+        this.pizzaCount = checkQuantityPizza(pizzaCount);
+    }
+
+        public int checkQuantityPizza(int pizzaCount) {
+        if (pizzaCount > 1  && pizzaCount <= 10) {
+            return pizzaCount;
+        }
+        System.out.println("From one pizza you can order no more than 10");
+        return 0;
+    }
+
+    public Pizza getPizza() {
+        return pizza;
+    }
+
+    public int getCount() {
+        return pizzaCount;
+    }
+
+    public static OrderItem addOrderItem(Pizza pizza, int pizzaCount) {
```

Comment:
Tatev everything is good, thank you. The only thing that I noticed these add... static methods, there is no need of these methods, just use constructors instead